omnigibson/object_states/attached_to.py

```diff
@@ -316,7 +316,7 @@ def _attach(self, other, child_link, parent_link, joint_type=None, can_joint_bre
             _, parent_local_quat = T.relative_pose_transform([0, 0, 0], child_quat, [0, 0, 0], parent_quat)
 
         # Disable collision between the parent and child objects
-        self._disable_collision_between_child_and_parent(child=self.obj, parent=other)
+        # self._disable_collision_between_child_and_parent(child=self.obj, parent=other)
```

Comment:
I am not entirely sure. I think this might be needed for other attachment assets (not needed for the shelves in our demo). In any case, I will leave the code here and do the refactoring after release.